.circleci/config.yml

```diff
@@ -885,8 +880,10 @@ jobs:
       - upload-test-results
 
   test-mac-arm64:
-    executor: mac-arm64
+    executor: python/default
+    resource_class: macos.m1.medium.gen1
     environment:
+      EMSDK_NOTTY: "1"
```

Comment:
What are these changes?

Reply:
This fix landed as #22260.  Reverted here.